src/responsive-news.js

```diff
@@ -0,0 +1,72 @@
+/*
+APIKEY - 4e6aa60e17534b93ad4407fa0a2256a7
+*
+*/
+/*
+*/
+const formEvent = document.querySelector("#form");
+
+formEvent.addEventListener("submit", function(e) {
+  e.preventDefault();
+  const cityName = document.querySelector("#input");
+
+  getNews(cityName.value);
+  getFlag(cityName.value);
+});
+
+function getNews(cityName) {
+  const sideBarImg = document.querySelector(".app__content-sidebar");
+  const bodyContent = document.querySelector(".app__content-body");
+  const rightNav = document.querySelector(".app__content-nav");
+  let apiCall = encodeURI(
+    `https://newsapi.org/v2/top-headlines?q=${cityName}&apiKey=4e6aa60e17534b93ad4407fa0a2256a7`
+  );
+  fetch(apiCall)
+    .then(function(response) {
+      return response.json();
+    })
+    .then(function(data) {
+      const articlesData = data;
+
+      let title = data.articles.map(function(item) {
```

Comment:
It would have been better here to create HTML snippets for each news item, rather than pull place items in separate lists. It may not be clear in the layout how the different pieces fit together